.github/copilot-instructions.md

```
@@ -0,0 +1,1079 @@
+Write towards the reader directly, and not a group of users. Write in a
+second-person perspective, using "you" and "your" instead of "the user"
+or "users".
+
+Make the text feel personal and friendly, as if you are talking to a friend who
+really enjoys technology and enjoys this hobby of home automation. Write in
+an informational and friendly tone, and not in a formal or technical tone;
+creating an informing, inspiring, personal, comforting, engaging, and
+welcoming experience for the reader.
+
+Some other rules:
+- Use the Oxford comma.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)